dll/win32/shlwapi/ordinal.c

```diff
@@ -5729,7 +5729,7 @@ HRESULT WINAPI SHPropertyBag_ReadGUID(IPropertyBag *ppb, LPCWSTR pszPropName, GU
     else if (V_VT(&vari) == VT_BSTR)
         bRet = GUIDFromStringW(V_BSTR(&vari), pguid);
     else
-#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
+#if (WINVER >= _WIN32_WINNT_VISTA)
```

Comment:
Huh why? (this is incorrect).

Reply:
It seems `_WIN32_WINNT` definition was incorrect.

Reply:
"Incorrect": well, it is redefined, for a reason : https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/shlwapi/CMakeLists.txt;h=af8ec781824f46c03d8c336f1a59b74cab4818b7;hb=HEAD#l3

Reply:
Any solution?

Reply:
Can you identify all the parts that do require it to be redefined? (The compiler should help when looking at the build issues when **locally** undefining it). Is it okay to split that question from your current PR?

Reply:
At least, `RegGetValueW` prototype is missing.

Reply:
Adding `RegGetValueW` prototype to `assoc.c` does fix this issue.

Reply:
https://github.com/reactos/reactos/pull/5514